docs/csharp/tutorials/exploration/csharp-6.yml

```
@@ -0,0 +1,240 @@
+### YamlMime:Tutorial
+title: Explore C# 6 - C# interactive C# tutorial
+metadata:
+  title: Explore C# 6 - Try the new features in C# 6 interactively, using your browser
+  description: In this tutorial, you'll use your browser to explore C# 6 interactively. You'll explore the new idioms you can use with C# 6 that enable more concise and readable code.
+  audience: Developer
+  level: intermediate
+  ms.date: 12/11/2018
+  displayType: two-column
+  interactive: csharp
+items:
+- durationInMinutes: 1
+  content: |
+    This tutorial lets you explore C# 6 interactively, using your browser to write C# and see the results of compiling and running your code. It contains a series of lessons that modify earlier C# practices to use newer, more concise C# 6 features. The rest of this article provides an overview of each of these features, with a link to explore each feature. You can read about the features in the [What's new in C# 6](../whats-new/csharp-6.md) article in the "What's new" section.
+
+- title: Read only auto-properties enable readonly types
```

```suggestion
- title: Read-only auto-properties enable read-only types
```

related: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/read-only